### ISSUE TITLE
my-first-react-jss-app doesn't work with url ip:port (Cannot read property 'sitecore' of null)

### ISSUE THREAD

**Issue by SteppeEagle** [Sitecore/jss#239]
## Description
You cannot run `my-first-react-jss-app` (react template) with url `ip:port` only with `localhost`.

For example if I run app on 10.110.6.130, I should be able to open application.

![running](https://user-images.githubusercontent.com/5242872/63002936-bb0fa700-be7f-11e9-8aca-365b9acc3873.png)
But I will get: Cannot read property 'sitecore' of null

![error](https://user-images.githubusercontent.com/5242872/63003056-0d50c800-be80-11e9-8df1-717db0baa1c8.png)

## Expected behavior
It should open app without errors or it should say in documentation that it can be run only on localhost or provide some workaround solution.

## Steps To Reproduce
Just follow documentation: https://jss.sitecore.com/docs/getting-started/quick-start

- npm install -g @sitecore-jss/sitecore-jss-cli
- jss create my-first-jss-app react
- cd my-first-jss-app
- jss start

## My uderstanding and suggestion
At the moment `localhost` is hardcoded in `src/setupProxy` and in `scripts/bootstrap`.
It leads to CORS error because url has `ip:port` but request go to `localhost`.
My solution was 
- change `localhost` to ip in both files, 
- pass --disconnected arg to  `src/setupProxy`
- change implementation of `isDisconnected` 
```
const isDisconnected = /localhost/i.test(config.sitecoreApiHost);
// to 
const isDisconnected = process.argv.some((arg) => arg === '--disconnected');
```
but drawback is that localhost url will not work.
## Your Environment

* Sitecore Version: - 
* JSS Version: 12.0.0
* Browser Name and version: Chrome 76.0.3809.100
* Operating System and version (desktop or mobile): Ubuntu 18.04 and Windows 10

**Comment by zfrankdesign** [comment]
Any quick fixes for this? I am also having the same problem.

**Comment by sc-illiakovalenko** [comment]
@SteppeEagle Fixed in #403
